plugins/labhub.py

```diff
@@ -333,7 +333,8 @@ def eligible(user, iss):
                            'issue. :tada:')
                 else:
                     yield 'You are not eligible to be assigned to this issue.'
-                    yield '\n'.join(eligility_conditions)
+                    yield '\n'.join(
+                        bility_conditions)
```

Comment:
🤔

Reply:
Wait how is this even pass tests?

Reply:
probably not being tested. That would be a separate issue though

Reply:
If it's not tested, wouldn't test coverage be < 100% and cause the code to fail CI?

Reply:
Yeah, how does this work? This should raise `NameError` :confused:

Reply:
autoflake isnt enabled?

Reply:
fyi, this was a merge conflict.  His original patch was good, but needed rebasing.